validator/src/bootstrap.rs

```diff
@@ -653,6 +653,15 @@ fn get_rpc_nodes(
             cluster_info,
             &rpc_peers,
             validator_config.known_validators.as_ref(),
+            if newer_cluster_snapshot_timeout
+                .as_ref()
+                .map(|timer: &Instant| timer.elapsed().as_secs())
+                < Some(60)
```

Comment:
> I saw it take 7-10 seconds before the node would get all the snapshot hashes from all the known validators

Wonder if we should cut bait earlier? On the other hand, waiting 60 seconds (as opposed to say 30) is still minor in comparison to the time to download and unpack snapshots, so maybe this isn't so bad. Especially since we'll return if we finish before this time elapses